spec/controllers/v1/damage_types_controller_spec.rb

```diff
@@ -0,0 +1,159 @@
+require 'rails_helper'
+
+describe V1::DamageTypesController do
+  let!(:creator) { authenticate! }
+  describe 'POST #create' do
+    let!(:name) { 'TEST DAMAGE TYPE' }
+    let(:submit) { post :create, name: name }
+
+    context 'damage type created successfully' do
+      let(:damage_type) { create :damage_type, creator_id: creator.id }
+      before :each do
+        expect(DamageType)
+          .to receive(:new)
+          .with(name: name, creator_id: creator.id)
+          .and_return damage_type
+      end
+      it 'has an ok status' do
+        submit
+        expect(response).to have_http_status :ok
+      end
+      it 'return the damage type' do
+        submit
+        expect(JSON.parse(response.body)).to eql('uuid' => damage_type.uuid,
+                                                 'name' => damage_type.name,
+                                                 'damages' => [])
+      end
+    end
+
+    context 'error when creating damage type' do
+      let(:name) { ' ' }
+      let(:error_messages) { ["Name can't be blank"] }
+      it 'has an unprocessable status' do
+        submit
+        expect(response).to have_http_status :unprocessable_entity
+      end
+
+      it 'returns the errors of the item type' do
+        submit
+        expect(JSON.parse(response.body)).to eql 'errors' => error_messages
+      end
+    end
+  end
+
+  describe 'POST #destroy' do
+    let(:submit) { delete :destroy, id: damage_type.uuid }
+    context 'damage type found' do
+      let(:damage_type) { create :damage_type }
+      it 'has an ok status' do
+        submit
+        expect(response).to have_http_status :ok
+      end
+      it 'delete any damages that belongs to the damage type' do
+        create :damage, damage_type: damage_type
+        expect(Damage.count).to be 1
```

Comment:
I think a `change` matcher is more concise here:

```ruby
expect { submit }.to change { Damage.count }.by(-1)
```

or if you prefer the specificity

```ruby
expect { submit }.to_change { Damage.count }.from(1).to 0
```